<!DOCTYPE html>
<html><meta http-equiv="Content-Type" content="text/html" charset="UTF-8" />
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />

<title>Berry Lab @ UNSW</title>


  <link rel="shortcut icon" href="https://berrygroup.github.io/images/favicon.png" type="image/png"
/>


  <meta name="description" content="Biological research lab based at the University of New South Wales, Sydney, Australia"
/>

  <meta name="keywords" content="transcription, berrygroup, berry lab, unsw, scaling, single molecule science, single cell, sydney, rna institute"
/>
<meta name="referrer" content="no-referrer-when-downgrade" />

<meta name="HandheldFriendly" content="True" />
<meta name="MobileOptimized" content="320" />
<meta name="viewport" content="width=device-width, initial-scale=1.0" />

<link rel="stylesheet" href="https://berrygroup.github.io/css/screen.css" />
<link
  href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300,300italic,700"
  rel="stylesheet"
  type="text/css"
/>
<link
  href="https://fonts.googleapis.com/css?family=Oswald:400,300,700|Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800|Roboto+Slab:400,100,300,700"
  rel="stylesheet"
  type="text/css"
/>
<link
  rel="stylesheet"
  href="https://berrygroup.github.io/fork-awesome/css/fork-awesome.min.css"
  type="text/css"
/>

<meta property="og:title" content="" />
<meta property="og:description" content="Biological research lab based at the University of New South Wales, Sydney, Australia" />
<meta property="og:type" content="website" />
<meta property="og:url" content="https://berrygroup.github.io/" /><meta property="og:image" content="https://berrygroup.github.io/images/cover-image.png" /><meta property="og:site_name" content="Berry lab - Quantitative regulation of gene expression in single cells" />


<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://berrygroup.github.io/images/cover-image.png" /><meta name="twitter:title" content=""/>
<meta name="twitter:description" content="Biological research lab based at the University of New South Wales, Sydney, Australia"/>

<meta itemprop="name" content="">
<meta itemprop="description" content="Biological research lab based at the University of New South Wales, Sydney, Australia">





<link href="https://fonts.googleapis.com/css2?family=Raleway:wght@400&display=swap" rel="stylesheet"> 
<link href="https://fonts.googleapis.com/css2?family=Fredoka%20One&display=swap" rel="stylesheet"> 
<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap" rel="stylesheet">

<style>

     h2,
     h3 {
          font-family: "Fredoka One";
          font-weight: 700;
          font-size: 3.5rem;
     }

     h1 {
          color:#f8fcfc;
          font-family: "Montserrat";
          font-weight: 700;

     }

      
     .blog-description {
          margin: 0;
          font-size: 2.5rem;
          line-height: 1.5em;
          font-weight: 400;
          font-family: "Raleway";
          letter-spacing: 0;
     }

     h3 {
          font-family: "Fredoka One";
          font-weight: 400;
          margin-top: 0px;
          margin-bottom: 0px;
     }

     h4 {
          font-family: "Montserrat";
          font-weight: 400;
          margin-top: 40px;
          margin-bottom: 20px;
     }

     h6 {
          font-family: "Montserrat";
          color:#afc9ce;
          font-weight: 400;
          font-size: 2.25rem;
          line-height: 1.25em;
          margin-top: 40px;
          margin-bottom: 0;
     }

      
     span {
          font-family: "Raleway";
          font-style: italic;
          font-weight: 400;
          font-size: 1.5rem;
          line-height: 1.25em;
          margin-top: 0px;
          margin-bottom: 0px;
          margin-right: 4em;
     }

     body {
          color: #32363f;
          font-family: "Raleway";
          font-weight: 400;
          font-size: 1.8rem;
     }

      
     a:hover {
         color: #b2a29a;
     }

      
     a.btn {
         color: #434248;
          background-color: #fae880;
     }
     a.btn:hover {
          background-color: #c4ba7f;
     }
     

      
     a.fn-item {
          color: #617d82;
          opacity: 1.0;
     }
     a.fn-item.active {
          color: #fae880;
     }
     a.fn-item:hover {
          color: #c4ba7f;
     }

      
     .post-holder.odd {
          background: #4f4990;
          color: #f8fcfc;
     }
     .post-holder {
          background: #f8fcfc;
     }

     .content-card img {
          max-width: 100%;
     }

      
     .site-footer {
          color: #617d82;
          background-color: #afc9ce;
     }
</style>









<link
  rel="apple-touch-icon"
  sizes="180x180"
  href="https://berrygroup.github.io/images/apple-touch-icon.png"
/>
<link
  rel="icon"
  type="image/png"
  sizes="32x32"
  href="https://berrygroup.github.io/images/favicon-32x32.png"
/>
<link
  rel="icon"
  type="image/png"
  sizes="16x16"
  href="https://berrygroup.github.io/images/favicon-16x16.png"
/>
<body>


<header id="site-head" style="background-image: url(images/header-background.png)">
    <div class="vertical">
        <div id="site-head-content" class="inner">

            <h1 class="blog-title">Quantitative regulation of gene expression in single cells</h1>
            <h2 class="blog-description">Berry Lab @ UNSW</h2>

            
                 <a class='btn site-menu' data-title-anchor='about-us'>About us</a>
            
                 <a class='btn site-menu' data-title-anchor='people'>People</a>
            
                 <a class='btn site-menu' data-title-anchor='research'>Research</a>
            
                 <a class='btn site-menu' data-title-anchor='publications'>Publications</a>
            
                 <a class='btn site-menu' data-title-anchor='contact'>Contact</a>
            
            <i id='header-arrow' class="fa fa-angle-down"></i>
        </div>
    </div>
</header>
<main class="content" role="main">

    <div class='fixed-nav'>
    </div>
    
        <div class='post-holder'>
            <article id='about-us' class='post first'>
                <header class="post-header">
                    <h2 class="post-title">About us</h2>
                </header>
                <section class="post-content">
                    <p>We are a research group at the <a href="https://www.rna.unsw.edu.au/">RNA Institute</a> and <a href="https://sms.unsw.edu.au/">Single Molecule Science</a>: the EMBL Australia node at the University of New South Wales. We are part of the <a href="https://www.unsw.edu.au/medicine-health/our-schools/biomedical-sciences/about-us/our-people/our-departments">Department of Molecular Medicine</a>. Please <a href="mailto:scott.berry@unsw.edu.au">contact Scott</a> if you are interested in joining or collaborating.</p>
<p><img src="images/UNSW_EMBL_RNA.png" alt="UNSW"></p>
<h2 id="positions-available">Positions available</h2>
<p>Please <a href="mailto:scott.berry@unsw.edu.au">email</a> if you are interested in a PhD or postdoc position in the lab.</p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='people' class='post '>
                <header class="post-header">
                    <h2 class="post-title">People</h2>
                </header>
                <section class="post-content">
                    <h4 id="scott-berry">Scott Berry</h4>
<p><img src="images/ProfilePicture.jpg#floatright" alt="Scott Berry"></p>
<h6 id="group-leader">Group leader</h6>
<p>Hi! I am Scott. After completing my undergraduate studies in Theoretical Physics and Biology at the University of Adelaide and the University of Western Australia, I moved to the UK for my PhD at the <a href="https://www.jic.ac.uk/">John Innes Centre</a>. There, I worked with <a href="https://www.jic.ac.uk/people/caroline-dean/">Caroline Dean</a> and <a href="https://www.jic.ac.uk/people/martin-howard/">Martin Howard</a>, combining theoretical and experimental biology approaches to understand mechanisms of epigenetic memory in <em>Arabidopsis thaliana</em>. I then moved to Switzerland as a <a href="https://www.hfsp.org/funding/hfsp-funding/postdoctoral-fellowships">HFSP postdoctoral fellow</a>, working with <a href="https://pelkmanslab.org/">Lucas Pelkmans</a> at the University of Zurich. In Lucas&rsquo; group, I shifted model systems from plants to mammalian cells, and became immersed in the wonderful world of modern single-cell biology &ndash; focusing on understanding the mechanisms by which cells coordinate gene expression with cell size. Since 2021, I have been a group leader at <a href="https://sms.unsw.edu.au/">Single Molecule Science</a> at the University of New South Wales, and also joined the <a href="https://www.rna.unsw.edu.au/">RNA Institute</a> in 2023.
<div style="clear: both;"></div></p>
<h4 id="alexander-gillis">Alexander Gillis</h4>
<p><img src="images/AlexGillis_BW.jpg#floatright" alt="Alex Gillis"></p>
<h6 id="postdoc">Postdoc</h6>
<p>I completed my undergraduate degree with majors in neuroscience and pharmacology. My PhD was undertaken at University of Sydney, working with Prof Macdonald Christie, during which I studied the pharmacology of opioid analgesics across in vitro assays and rodent behavioural studies. In the course of that work, I became interested in mathematical modelling of biological systems as a tool to unpick signalling mechanisms. As part of my PhD, I completed a research period at University of Nottingham, working with Prof Meritxell Canals. There, we sought to examine the kinetics of opioid receptor signalling on a population &amp; single molecule level. As a result of this approach to receptor signalling systems, I have become increasingly interested in using similar techniques to systematically deconstruct other biological pathways.</p>
<p><i class="fa fa-envelope"></i> <a href="mailto:a.gillis@unsw.edu.au">a.gillis[at]unsw.edu.au</a></p>
<p><i class="fa fa-twitter"></i> <a href="http://twitter.com/acftbl">@acftbl</a>
<div style="clear: both;"></div></p>
<h4 id="angela-arabiotorre">Angela Arabiotorre</h4>
<p><img src="images/Angela_BW.jpg#floatright" alt="Angela Arabiotorre"></p>
<h6 id="postdoc-1">Postdoc</h6>
<p>Howdy! I obtained my bachelor’s degree in Bioanalysis at Central University of Venezuela and was awarded the Outstanding Undergraduate Project certificate for studying molecular diagnostic techniques of Trypanosoma cruzi. I obtained my master’s degree in Molecular Biology and Biomedicine at University of the Basque Country (UPV/EHU) in Spain. At a research placement at Prof Banafshé Larijani’s laboratory in the Biophysics Institute (UPV/EHU), we analysed the phosphoinositide and protein physical proximity through the implementation of FRET microscopy to study the biogenesis of nuclear envelope assembly. I completed my Ph.D. candidacy under the guidance of Prof Vytas Bankaitis and Dr. Aby Grabon at Texas A&amp;M University (USA). There, I described the regulatory role of Phosphatidylinositol-4-phosphate in the dense granule biogenesis and exocytosis in Toxoplasma gondii. I am interested in studying the physiology and biochemistry of eukaryotic cells, including gene expression and inheritance, and how these were shaped through evolution.</p>
<p><i class="fa fa-envelope"></i> <a href="mailto:a.arabiotorre@unsw.edu.au">a.arabiotorre[at]unsw.edu.au</a></p>
<p><i class="fa fa-twitter"></i> <a href="http://twitter.com/acftbl">@acftbl</a>
<div style="clear: both;"></div></p>
<h4 id="teddy-cheng-wei-kang">Teddy Cheng Wei Kang</h4>
<p><img src="images/Teddy_BW.jpg#floatright" alt="Teddy Cheng Wei Kang"></p>
<h6 id="phd-student">PhD student</h6>
<p>Hello, I am Teddy. After completing my undergraduate in Resource Biotechnology at University of Malaysia, Sarawak, I continued my MSc in Molecular Medicine under Dr Oon Chern Ein at Universiti Sains Malaysia. This project involved collaborative work together with researchers at the Karolinska Institutet, aimed at determining if manic fringe protein expression can be utilized as a biomarker for therapeutics and prognosis in kidney cancer, as well as to determine its association with immune cell infiltration. Our research group also collaborated with the Weatherall Institute of Molecular Medicine to further elucidate the expression pattern of manic fringe in blood vessels of renal cancer and in tumor angiogenesis. I am currently a PhD student researching the global control of mRNA metabolism in human cells.</p>
<p><i class="fa fa-envelope"></i> <a href="mailto:c.wei_kang@unsw.edu.au">c.wei_kang[at]unsw.edu.au</a></p>
<h4 id="hongpei-li">Hongpei Li</h4>
<p><img src="images/Hongpei_BW.jpg#floatright" alt="Hongpei Li"></p>
<h6 id="phd-student-1">PhD student</h6>
<p>Hello, my name is Hongpei. I completed my undergraduate study in Biological Sciences at Jinan University in China, where I developed a strong interest in synthetic biology. I continued to pursue an MSc in synthetic biology and biotechnology at the University of Edinburgh, where I investigated the sugar consumption strategy in budding yeast using quantitative methods under the supervision of Professor Peter Swain. After completing my MSc, I worked as an R&amp;D trainee at Ailurus Biotechnology, developing a bio-computing platform based on understanding artificial biological processes in E.coli using massively parallel reporter assays. I joined Scott&rsquo;s group in 2023 as a PhD student and my current research focuses on understanding quantitative epigenetic memory in human cells using synthetic biology tools.</p>
<p><i class="fa fa-envelope"></i> <a href="mailto:hongpei.li@unsw.edu.au">hongpei.li[at]unsw.edu.au</a></p>
<h4 id="bailey-hiles-murison">Bailey Hiles-Murison</h4>
<p><img src="images/Bailey_BW.jpg#floatright" alt="Bailey Hiles-Murison"></p>
<h6 id="phd-student-2">PhD student</h6>
<p>Hi! My name is Bailey. I completed my undergraduate degree in Molecular Genetics and Biotechnology at Curtin University in 2021, studying honours under Ryu Takechi at Curtin’s Health Innovation and Research Institute. For my honours project I investigated the effects of Probucol on attenuating neurodegenerative changes associated with APOE4 genotype, using both in-vivo and ex-vivo analyses of transgenic mice expressing human APOE isoforms. I also worked on a murine model of traumatic brain injury where we established early neuropathological developments in the context of repetitive sub-concussion. Since beginning my studies I’ve always held a keen interest in cell biology, and I joined Scott’s group in 2023 as a PhD candidate, focusing on visualisation of transcription and mRNA processing in living cells.</p>
<p><i class="fa fa-envelope"></i> <a href="mailto:b.hilesmurison@unsw.edu.au">b.hilesmurison[at]unsw.edu.au</a></p>
<p><i class="fa fa-twitter"></i> <a href="http://twitter.com/blymrsn">@blymrsn</a>
<div style="clear: both;"></div></p>
<h4 id="open-positions">Open positions</h4>
<h6 id="postdoc--phd--honours">Postdoc / PhD / Honours</h6>
<p>Please <a href="mailto:scott.berry@unsw.edu.au">contact Scott</a> if you are interested in applying. More information on scholarships for Australian and international students can be found <a href="https://research.unsw.edu.au/graduate-research-scholarships">here</a>.</p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='research' class='post '>
                <header class="post-header">
                    <h2 class="post-title">Research</h2>
                </header>
                <section class="post-content">
                    <p>We are interested in the mechanisms used by cells to regulate gene-specific and global RNA abundance &ndash; ranging from <strong>epigenetic memory</strong> and cellular decision-making to <strong>global RNA metabolism</strong> in the context of cellular physiology. We seek to generate a <em>quantitative</em> understanding of these processes.</p>
<p>Our research methodology is a combination of &rsquo;top-down&rsquo; (data-driven) and &lsquo;bottom-up&rsquo; (hypothesis-driven) approaches. We make extensive use of <strong>high-throughput microscopy</strong> and automated image analysis, which enables detailed measurement of quantitative cellular phenotypes (for example, the abundance and localisations of specific proteins or RNAs) across large cell populations. Resulting datasets are then analysed using a variety of methods from <strong>data science</strong>. We complement these image-based approaches with <strong>functional genomics</strong> experiments, based on next-generation sequencing &ndash; providing exquisite genomic resolution, and use perturbation experiments including <strong>genome and epigenome-editing</strong> to test specific hypotheses.</p>
<p><img src="images/methodology_schematic-01.png" alt="Effects on gene expression of CRISPR-KO using smFISH"></p>
<p>Projects in the lab are often underpinned by construction of minimal <strong>mathematical models</strong> at the molecular level, which we use as a tool for exploring possible explanations of experimental results and prioritising future experiments.</p>
<p>We believe that the best science happens in the overlap between fields. We strive to build a diverse team and foster collaborations across fields and disciplines.</p>
<h4 id="stability-and-plasticity-of-epigenetic-memory">Stability and plasticity of epigenetic memory</h4>
<p>The chromatin of individual genetic loci can be set into molecular &lsquo;states&rsquo; that control the expression of the underlying genes, and in some cases even instruct their own inheritance &ndash; acting as local epigenetic memory elements. When necessary, however, chromatin can also be reliably switched from one state to another &ndash; by processes that involve coordinated changes to the histone modifications, accessibility of regulatory sequences, and expression states of the underlying genes.</p>
<p>We are interested in the interplay between gene-specific diffusible regulatory factors (e.g. transcription factors), which tend to have continuous (analogue) control over gene expression, and heritable chromatin states (e.g. Polycomb-repressed chromatin), which tend to promote bi-modal (digital) stable expression states. To study these processes we use in vitro differentiation of epithelial cells from human induced pluripotent stem cells.</p>
<p><strong>Further reading:</strong><br>
<a href="http://dx.doi.org/10.1126/science.aan1121">Yang, Berry <em>et al.</em>, Science, 2017</a><br>
<a href="http://dx.doi.org/10.1016/j.cels.2017.02.013">Berry <em>et al.</em>, Cell Systems, 2017</a><br>
<a href="http://dx.doi.org/10.7554/eLife.79743">Antoniou-Kourounioti <em>et al.</em>, eLife, 2023</a></p>
<p><img src="images/Modelling-01.png" alt="Integrating local and diffusible signals at chromatin">
<span> 
Mechanistic model of self-perpetuating active and repressed chromatin states. 
 </span></p>
<h4 id="global-control-of-rna-metabolism">Global control of RNA metabolism</h4>
<p>Mammalian cells coordinate RNA production rates with cell size to ensure RNA concentration homeostasis. We discovered that this process relies on adapting transcription rates according to nuclear mRNA concentration, so that overabundance of nuclear mRNA leads to transcriptional repression. This mechanism involves mRNA-based regulation of the activity of RNA Polymerase II, which in turn determines polymerase abundance. We are interested in further dissecting the interplay between the transcriptional machinery, nuclear RNA export and RNA degradation factors that act in concert to ensure robust mRNA concentration homeostasis.</p>
<p><strong>Further reading:</strong><br>
<a href="https://doi.org/10.1016/j.cels.2022.04.005">Berry <em>et al.</em>, Cell Systems, 2022</a><br>
<a href="https://doi.org/10.1016/j.tcb.2022.05.003">Berry <em>et al.</em>, Trends in Cell Biology, 2022</a></p>
<p><img src="images/transcription_feedback.png" alt="Feedback from RNA production"></p>
<h4 id="high-dimensional-cellular-phenotyping">High-dimensional cellular phenotyping</h4>
<p><img src="images/4i.gif#floatrighthalf" alt="4i">
In recent years, technological advances have made it possible to visualise tens to hundreds of proteins in single cells. This has brought with it the ability to accurately quantify the high-dimensional relationships between protein abundances and their modification states, as well as the cell-to-cell variability of their subcellular localisation and co-localisation.</p>
<p>We use <a href="https://doi.org/10.1126/science.aar7042">Iterative Indirect Immunofluorescence Imaging</a> (4i) throughout our research to comprehensively evaluate changes to cellular phenotypes in the context of cellular differentiation and upon perturbation of RNA metabolism. These datasets are extraordinarily rich in information, however much of this information is not easy to interpret using existing approaches. In collaboration with <a href="https://www.pelkmanslab.org">Lucas Pelkmans&rsquo;</a> and <a href="https://www.helmholtz-muenchen.de/icb/research/groups/theis-lab/overview/index.html">Fabian Theis&rsquo;</a> research groups, we have developed a new computational image analysis workflow called CAMPA to facilitate analysis of these data across different perturbations, at the subcellular, cellular and population scales.</p>
<p><strong>Further reading:</strong><br>
<a href="https://doi.org/10.1038/s41592-023-01894-z">Spitzer, Berry <em>et al.</em>, Nat Methods, 2023</a><br>
<a href="https://github.com/theislab/campa">CAMPA on GitHub</a></p>
<p><img src="images/CSL-01.jpg" alt="Consistent subcellular landmarks"></p>
<span> 
Applying pixel-clustering approaches to highly-multiplexed 4i image data identifies subcellular structures in an unbiased manner. In this case, we focused on the nucleus: revealing organelles such as nuclear speckles, Cajal bodies, PML bodies and the nucleolus.
 </span>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='publications' class='post '>
                <header class="post-header">
                    <h2 class="post-title">Publications</h2>
                </header>
                <section class="post-content">
                    <hr>
<h3 id="preprints">Preprints</h3>
<h6 id="robust-and-versatile-arrayed-libraries-for-human-genome-wide-crispr-activation-deletion-and-silencing">Robust and Versatile Arrayed Libraries for Human Genome-Wide CRISPR Activation, Deletion and Silencing</h6>
<p>Yin, J.-A., Scheidmann, M. C., Frick, L., Trevisan, C., Dhingra, A., Spinelli, A., Wu, Y., Yao, L., Vena, D.L., De Cecco, E., Ging, K., Liu, T., Täger, J., Rodriguez, S., Guo, J., Berry, S., Losa, M., Hornemann, S., Kampmann, M., Pelkmans, L., Hoepfner, D., Heutink, P., and Aguzzi, A.<br>
<em>bioRxiv</em>
| <a href="https://www.biorxiv.org/content/10.1101/2022.05.25.493370v1.full.pdf">PDF</a>
| <a href="https://www.biorxiv.org/content/10.1101/2022.05.25.493370">bioRxiv</a></p>
<hr>
<h3 id="2024">2024</h3>
<h6 id="global-control-of-rna-polymerase-ii">Global control of RNA polymerase II</h6>
<p>Gillis, A., and Berry, S.<br>
<em>BBA Gene Regulatory Mechanisms</em>
| <a href="papers/gillis_bba_2024.pdf">PDF</a>
| <a href="https://doi.org/10.1016/j.bbagrm.2024.195024">Journal</a></p>
<hr>
<h3 id="2023">2023</h3>
<h6 id="integrating-analog-and-digital-modes-of-gene-expression-at-arabidopsis-flc">Integrating analog and digital modes of gene expression at Arabidopsis <em>FLC</em></h6>
<p>Antoniou-Kourounioti, R.L.*, Meschichi, A.*, Reeck, S.*, Berry, S., Menon, G., Zhao, Y., Fozard, J., Holmes, T., Zhao, L., Wang, H., Hartley, M., Dean, C., Rosa, S., Howard, M.
<em>eLife</em>
| <a href="papers/antoniou-kourounioti_et_al_elife_2023.pdf">PDF</a>
| <a href="https://doi.org/10.7554/eLife.79743">Journal</a>
| <a href="https://doi.org/10.1101/2022.07.04.498694">bioRxiv</a><br>
* Equal contribution</p>
<h6 id="learning-consistent-subcellular-landmarks-to-quantify-changes-in-multiplexed-protein-maps">Learning consistent subcellular landmarks to quantify changes in multiplexed protein maps</h6>
<p>Spitzer, H.*, Berry, S.*, Donoghoe, M., Pelkmans, L. and Theis, F.<br>
<em>Nature Methods</em>
| <a href="papers/spitzer_et_al_nat_methods_2023.pdf">PDF</a>
| <a href="https://doi.org/10.1038/s41592-023-01894-z">Journal</a>
| <a href="https://doi.org/10.1038/s41592-023-01895-y">Research Briefing</a>
| <a href="https://www.biorxiv.org/content/10.1101/2022.05.07.490900">bioRxiv</a>
| <a href="https://github.com/theislab/campa">GitHub</a>
| <a href="https://campa.readthedocs.io/en/stable/">Read the Docs</a><br>
* Equal contribution</p>
<hr>
<h3 id="2022">2022</h3>
<h6 id="mechanisms-of-cellular-mrna-transcript-homeostasis">Mechanisms of cellular mRNA transcript homeostasis</h6>
<p>Berry, S., and Pelkmans, L.<br>
<em>Trends in Cell Biology</em>
| <a href="https://doi.org/10.1016/j.tcb.2022.05.003">Journal</a></p>
<h6 id="feedback-from-nuclear-rna-on-transcription-promotes-robust-rna-concentration-homeostasis-in-human-cells">Feedback from nuclear RNA on transcription promotes robust RNA concentration homeostasis in human cells</h6>
<p>Berry, S.^, Müller, M., Rai, A. and Pelkmans, L.^<br>
<em>Cell Systems</em>
| <a href="papers/berry_et_al_cell_systems_2022.pdf">PDF</a>
| <a href="https://doi.org/10.1016/j.cels.2022.04.005">Journal</a>
| <a href="https://doi.org/10.1101/2021.05.17.444432">bioRxiv</a><br>
^ Corresponding authors</p>
<hr>
<h3 id="2021">2021</h3>
<h6 id="high-content-genome-wide-sirna-screen-to-investigate-the-coordination-of-cell-size-and-rna-production">High content genome-wide siRNA screen to investigate the coordination of cell size and RNA production</h6>
<p>Müller, M., Avar, M., Heinzer, D., Emmenegger, M., Aguzzi, A., Pelkmans, L.^, and Berry, S.^<br>
<em>Scientific Data</em>
| <a href="papers/mueller_et_al_sci_data_2021.pdf">PDF</a>
| <a href="http://dx.doi.org/10.1038/s41597-021-00944-5">Journal</a>
| <a href="https://doi.org/10.17867/10000157">Data</a><br>
^ Corresponding authors</p>
<hr>
<h3 id="2017">2017</h3>
<h6 id="disruption-of-an-rna-binding-hinge-region-abolishes-lhp1-mediated-epigenetic-repression">Disruption of an RNA-binding hinge region abolishes LHP1-mediated epigenetic repression</h6>
<p>Berry, S., Rosa, S., Howard, M., Bühler, M., and Dean, C.<br>
<em>Genes &amp; Development</em>
| <a href="papers/berry_et_al_genes_dev_2017.pdf">PDF</a>
| <a href="http://dx.doi.org/10.1101/gad.305227.117">Journal</a></p>
<h6 id="slow-chromatin-dynamics-allow-polycomb-target-genes-to-filter-fluctuations-in-transcription-factor-activity">Slow Chromatin Dynamics Allow Polycomb Target Genes to Filter Fluctuations in Transcription Factor Activity</h6>
<p>Berry, S., Dean, C., and Howard, M.<br>
<em>Cell Systems</em>
| <a href="papers/berry_et_al_cell_systems_2017.pdf">PDF</a>
| <a href="papers/berry_et_al_cell_systems_2017_supplementary.pdf">Supplementary</a>
| <a href="http://dx.doi.org/10.1016/j.cels.2017.02.013">Journal</a>
| <a href="https://doi.org/10.1016/j.cels.2017.04.005">Commentary</a></p>
<h6 id="distinct-phases-of-polycomb-silencing-to-hold-epigenetic-memory-of-cold-in-arabidopsis">Distinct phases of Polycomb silencing to hold epigenetic memory of cold in Arabidopsis</h6>
<p>Yang, H.*, Berry, S.*, Olsson, T.S.G., Hartley, M., Howard, M., and Dean, C.<br>
<em>Science</em>
| <a href="papers/yang_et_al_science_2017.pdf">PDF (accepted version)</a>
| <a href="http://dx.doi.org/10.1126/science.aan1121">Journal</a><br>
* Equal contribution</p>
<hr>
<h3 id="2015">2015</h3>
<h6 id="local-chromatin-environment-of-a-polycomb-target-gene-instructs-its-own-epigenetic-inheritance">Local chromatin environment of a Polycomb target gene instructs its own epigenetic inheritance</h6>
<p>Berry, S., Hartley, M., Olsson, T.S.G., Dean, C., and Howard, M.<br>
<em>eLife</em>
| <a href="papers/berry_et_al_elife_2015.pdf">PDF</a>
| <a href="http://dx.doi.org/10.7554/elife.07205">Journal</a></p>
<h6 id="environmental-perception-and-epigenetic-memory-mechanistic-insight-through-flc">Environmental perception and epigenetic memory: mechanistic insight through <em>FLC</em></h6>
<p>Berry, S., and Dean, C.<br>
<em>The Plant Journal</em>
| <a href="papers/berry_and_dean_plant_journal_2015.pdf">PDF</a>
| <a href="http://dx.doi.org/10.1111/tpj.12869">Journal</a></p>
<hr>
<h3 id="2014">2014</h3>
<h6 id="lipopolysaccharide-o-antigen-size-distribution-is-determined-by-a-chain-extension-complex-of-variable-stoichiometry-in-escherichia-coli-o9a">Lipopolysaccharide O antigen size distribution is determined by a chain extension complex of variable stoichiometry in Escherichia coli O9a</h6>
<p>King, J.D.*, Berry, S.*, Clarke, B.R., Morris, R.J., and Whitfield, C.<br>
<em>Proceedings of the National Academy of Sciences</em>
| <a href="papers/king_et_al_pnas_2014.pdf">PDF</a>
| <a href="papers/king_et_al_pnas_2014_supplementary.pdf">Supplementary</a>
| <a href="http://dx.doi.org/10.1073/pnas.1400814111">Journal</a><br>
* Equal contribution</p>
<hr>
<h3 id="2012">2012</h3>
<h6 id="illustrations-of-mathematical-modeling-in-biology-epigenetics-meiosis-and-an-outlook">Illustrations of Mathematical Modeling in Biology: Epigenetics, Meiosis, and an Outlook</h6>
<p>Richards, D.*, Berry, S.*, and Howard, M.<br>
<em>Cold Spring Harbor Symposia on Quantitative Biology</em>
| <a href="papers/richards_et_al_cshs_2012.pdf">PDF</a>
| <a href="http://dx.doi.org/10.1101/sqb.2013.77.015941">Journal</a><br>
* Equal contribution</p>
<hr>
<h3 id="2011">2011</h3>
<h6 id="qwviz-visualisation-of-quantum-walks-on-graphs">qwViz: Visualisation of quantum walks on graphs</h6>
<p>Berry, S.D., Bourke, P., and Wang, J.B.<br>
<em>Computer Physics Communications</em>
| <a href="http://dx.doi.org/10.1016/j.cpc.2011.06.002">Journal</a></p>
<h6 id="two-particle-quantum-walks-entanglement-and-graph-isomorphism-testing">Two-particle quantum walks: Entanglement and graph isomorphism testing</h6>
<p>Berry, S.D., and Wang, J.B.<br>
<em>Physical Review A</em>
| <a href="http://dx.doi.org/10.1103/physreva.83.042317">Journal</a></p>
<!-- raw HTML omitted -->
<hr>
<h3 id="2010">2010</h3>
<h6 id="quantum-walk-based-search-and-centrality">Quantum-walk-based search and centrality</h6>
<p>Berry, S.D., and Wang, J.B.<br>
<em>Physical Review A</em>
| <a href="https://arxiv.org/abs/1010.0764">arXiv</a>
| <a href="http://dx.doi.org/10.1103/physreva.82.042333">Journal</a></p>
<!-- raw HTML omitted -->

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='contact' class='post last'>
                <header class="post-header">
                    <h2 class="post-title">Contact</h2>
                </header>
                <section class="post-content">
                    <p><i class="fa fa-phone"></i> <a href="tel:+61293480790">+61 2 93480790</a></p>
<p><i class="fa fa-envelope"></i> <a href="mailto:scott.berry@unsw.edu.au">scott.berry[at]unsw.edu.au</a></p>
<p><i class="fa fa-twitter"></i> <a href="http://twitter.com/scottbscience">@scottbscience</a></p>
<p><i class="fa fa-map-marker"></i> <a href="https://goo.gl/maps/sXyHpDVg2CQNv31P8">Lowy Cancer Research Centre</a></p>
<hr>
<p><em><strong>Find us:</strong></em></p>
<p>Single Molecule Science<br>
Room 311 (Level 3) Lowy Cancer Research Centre<br>
Cnr Botany &amp; High Sts, Building C25<br>
UNSW Kensington Campus</p>
<p><em><strong>Send something:</strong></em></p>
<p>Attention: Scott Berry<br>
Single Molecule Science<br>
Upper Campus Store E26, Bioscience South<br>
LG018 Loading Dock<br>
Via Gate 11, Botany Street<br>
UNSW Sydney NSW 2052<br>
Australia</p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
</main>
<footer class="site-footer">
  <div class="inner">
    
      <section class="copyright">© Scott Berry</section>


    <section></section>
    

  </div>
</footer>

    <script
      type="text/javascript"
      src="https://code.jquery.com/jquery-1.11.3.min.js"
    ></script>

    
    <script type="text/javascript" src='https://berrygroup.github.io/js/icons.js'></script>
    <script type="text/javascript" src='https://berrygroup.github.io/js/index.js'></script>
    
</body>
</html>
